.changeset/popular-apples-share.md

```diff
@@ -0,0 +1,8 @@
+---
+'@crackle/babel-plugin-remove-exports': minor
+'@crackle/cli': minor
+'@crackle/core': minor
+'@crackle/router': minor
```

Comment:
I feel like a dependency update should just be a fix since there's no new API.

```suggestion
'@crackle/babel-plugin-remove-exports': patch
'@crackle/cli': patch
'@crackle/core': patch
'@crackle/router': patch
```

Reply:
Luckily, it's addressed in the [spec](https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api).